rakefile.rb

```diff
@@ -31,6 +31,18 @@
 CLEAN.include(FileList["src/**/#{CONFIGURATION}"])
 CLEAN.include(FileList["test/**/#{CONFIGURATION}"])
 
+desc "Restore NuGet packages"
+exec :nuget_restore do |cmd|
+   cmd.command = "tools/nuget/NuGet.exe"
+   cmd.parameters = ["restore #{SOLUTION_FILE}"]
+end
+
+desc "Restore NuGet packages on Mono"
+exec :nuget_restore_mono do |cmd|

```

Comment:
@thecodejunkie You didn't add this mono task in the other repos. I assume we don't use them anyway?